x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/FunctionRegistry.java

```diff
@@ -95,16 +95,17 @@
             def(Kurtosis.class, Kurtosis::new),
         // Scalar functions
             // Date
-            def(DayOfMonth.class, DayOfMonth::new, "DAY", "DOM"),
-            def(DayOfWeek.class, DayOfWeek::new, "DOW"),
-            def(DayOfYear.class, DayOfYear::new, "DOY"),
-            def(HourOfDay.class, HourOfDay::new, "HOUR"),
-            def(MinuteOfDay.class, MinuteOfDay::new),
-            def(MinuteOfHour.class, MinuteOfHour::new, "MINUTE"),
-            def(SecondOfMinute.class, SecondOfMinute::new, "SECOND"),
-            def(MonthOfYear.class, MonthOfYear::new, "MONTH"),
-            def(Year.class, Year::new),
-            def(WeekOfYear.class, WeekOfYear::new, "WEEK"),
```

Comment:
because all sql date functions now take more than one parameter, had to change definition to match this new detail.